bodhi/server/models.py

```diff
@@ -3008,11 +3008,16 @@ def comment(self, session, text, karma=0, author=None, anonymous=False,
         if not author:
             raise ValueError('You must provide a comment author')
 
-        caveats = []
+        got_feedback = False
+        for feedback_dict in (bug_feedback + testcase_feedback):
+            if feedback_dict['karma'] != 0:
+                got_feedback = True
+                break
```

Comment:
IIRC, when I discovered #2623 last week I found that ```bug_feedback``` contains comments that are not part of the current request. I don't believe the code above will solve that particular problem.

Reply:
I have reverted back to the initial commit. 
About the second commit, I've tested it with the example update you suggested and it was working as expected. I will have a second look at that.

Reply:
For example, the bug_feedback passed for that update is
`u'bug_feedback': [{u'bug_id': 1618561, u'comment_id': 841956, u'bug': {u'bug_id': 1618561, u'security': False, u'parent': False, u'title': u'libssh_threads.so.4: cannot open shared object file'}, u'karma': 0}]`
It does not contain previous feedbacks